examples/window-app/common/window-app.matter

```diff
@@ -1891,10 +1940,22 @@ endpoint 1 {
     callback attribute serverList;
     callback attribute clientList;
     callback attribute partsList;
+    callback attribute generatedCommandList;
+    callback attribute acceptedCommandList;
+    callback attribute attributeList;
     ram      attribute featureMap;
     callback attribute clusterRevision default = 1;
   }
 
+  server cluster Binding {
```

Comment:
Why is this here?  I don't see Binding involved in this device type at all.

Reply:
https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/device_types/WindowCovering.adoc

Tell me if i am wrong
but this device type is classified as Simple
I was a bit lost with the requirements for Simple device there
https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/device_types/BaseDeviceType.adoc
Normally i don't need it since it is for `Simple and Client`
Can you just confirm that neither Binding Client or Server are needed @bzbarsky-apple ? thanks

Reply:
You need a Binding server for a device type that:
1. Is Simple
2. Includes a client cluster.

because then we need some way to tell that client cluster which server(s) to talk to.  I'd really like to understand what the confusing part here was so we can fix it in the spec to be clearer.

The Window Covering device does not have any client clusters, so does not need a Binding server.  And it definitely does not need a binding Client: that's for the device to tell other devices who to talk to, and Window Covering is not going to be doing that, right?

So no, you don't need either side of Binding here @jmeg-sfy

Reply:
Thanks for clarification ! I ll update